src/bin/pgcopydb/ld_transform.c

```diff
@@ -597,8 +601,17 @@ stream_transform_send_stop(Queue *queue)
  * target database.
  */
 bool
-stream_transform_file(char *jsonfilename, char *sqlfilename)
+stream_transform_file(StreamSpecs *specs, char *jsonfilename, char *sqlfilename)
```

Comment:
Given previous comment, I believe we should be able to remove all the changes here.